### ISSUE TITLE
Add accessibility lints to CI

### ISSUE THREAD

**Issue by TimJentzsch** [bevyengine/bevy-website#396]
I'm not sure if there is an easy way to add this to Zola projects, but it would be nice to have basic accessibility checks in CI.
Stuff like "every `img` element has an `alt` attribute" should be enough for the start.

If Zola has anything built-in we could use that, otherwise maybe it's possible to do the linting on the generated HTML?

**Comment by TrialDragon** [comment]
Did a little bit of research on this. [`pa11y`](https://github.com/pa11y/pa11y) seems to be a good candidate and has a version explicitly for CI: [`pa11y-ci`](https://github.com/pa11y/pa11y-ci).

I've done a local test with the tool and once installed it does work to catch accessibility issues on the site; technically more than even the Firefox accessibility tools showed me. Showed missing image alt text and told me the recommended color to use for each contrast issue.
